kaiser.gemspec

```diff
@@ -15,16 +15,6 @@ Gem::Specification.new do |spec|
   spec.homepage      = 'https://github.com/kaiser'
   spec.license       = 'MIT'
 
-  # Prevent pushing this gem to RubyGems.org. To allow pushes either set the
-  # 'allowed_push_host' to allow pushing to a single host or delete this section
-  # to allow pushing to any host.
-  if spec.respond_to?(:metadata)
-    spec.metadata['allowed_push_host'] = 'https://gems.labs.degica.com'
```

Comment:
Actually you just have to change this to 'https://rubygems.org'

and then run `make release`